packages/platform/src/internal/fetchHttpClient.ts

```diff
@@ -15,7 +16,7 @@ const fetch: Client.HttpClient = client.make((request, url, signal, fiber) => {
   const context = fiber.getFiberRef(FiberRef.currentContext)
   const fetch: typeof globalThis.fetch = context.unsafeMap.get(fetchTagKey) ?? globalThis.fetch
   const options: RequestInit = context.unsafeMap.get(requestInitTagKey) ?? {}
-  const headers = new globalThis.Headers(request.headers)
+  const headers = Headers.merge(request.headers, Headers.fromInput(options.headers))
```

Comment:
Not sure if this is the appropriate order to merge in, but this made sense to me

Reply:
The request headers should take precedence.